src/Storages/StorageBuffer.h

```diff
@@ -157,7 +157,7 @@ friend class BufferBlockOutputStream;
     /// `table` argument is passed, as it is sometimes evaluated beforehand. It must match the `destination`.
     void writeBlockToDestination(const Block & block, StoragePtr table);
 
-    void flushBack();
```

Comment:
I renamed this method, it was too difficult to read:
the name was similar to "flashback" that leads to confusion.